rust/src/lib.rs

```diff
@@ -1771,14 +1771,14 @@ to_from_bytes!(NativeScript);
 #[derive(Clone, Debug, Eq, Ord, PartialEq, PartialOrd)]
 pub enum ScriptHashNamespace {
```

Comment:
This is used on the client side, including Yoroi itself. I am double-checking atm that it will not cause runtime issues in JS and will decide then.

Reply:
@lisicky , so we are not removing this from bindgen for now, for the reason that it could cause runtime fails in JS as the field would become undefined. The changed `NativeScript.hash()` that now does not require the namespace argument does NOT cause runtime issues in old JS code when that namespace is being passed. The namespace can be removed from public access later in some major version